inventory-generation/notifications/main.yaml

```diff
@@ -55,7 +55,7 @@
 
       vars:
         ocp_subdomain: "{{ hosting_environments[0].ocp_sub_domain | lower }}"
-        subdomain_base_suffix: "{{ engagement_region | default('dev') + cloud_provider_index | default('-1') + ocp_base_url }}"
+        subdomain_base_suffix: "{{ engagement_region | default('dev') + cloud_provider_index | default('-1') + '.' + ocp_base_url }}"
```

Comment:
From what I gather, I don't think `ocp_base_url` is passed in anymore, so this code will never execute as-is. However, `subdomain_base_suffix` is passed in, so chances are we can probably simplify this code quite a bit with the latest changes.

Reply:
The notification inventory generation is still managed by the `resource-dispatcher` and we need `ocp_base_url` passed as external variable. The `subdomain_base_suffix` is passing only for playbooks executed by the Babylon operators and not available for `resource-dispatcher`.